examples/fuse_examples/imaging/classification/cmmd/runner.py

```diff
@@ -298,57 +294,41 @@ def run_eval(paths: dict, eval_common_params: dict):
 
     # run
     results = evaluator.eval(ids=None,
-                     data=os.path.join(paths["inference_dir"], eval_common_params["infer_filename"]),
+                     data=os.path.join(paths["inference_dir"], infer["infer_filename"]),
                      metrics=metrics,
-                     output_dir=paths['eval_dir'])
+                     output_dir=paths["eval_dir"])
 
     return results
-######################################
-# Run
-######################################
-
 
-if __name__ == "__main__":
-    # allocate gpus
-    if NUM_GPUS == 0:
-        TRAIN_COMMON_PARAMS['manager.train_params']['device'] = 'cpu'
+@hydra.main(config_path="conf", config_name="config")
+def main(cfg : DictConfig) -> None:
+    cfg = NDict(OmegaConf.to_container(cfg))
+    cfg["paths"]["data_dir"] = os.environ["CMMD_DATA_PATH"]
+    print(cfg)
     # uncomment if you want to use specific gpus instead of automatically looking for free ones
     force_gpus = None  # [0]
-    choose_and_enable_multiple_gpus(NUM_GPUS, force_gpus=force_gpus)
+    choose_and_enable_multiple_gpus(cfg["train.manager_train_params.num_gpus"], force_gpus=force_gpus)
+
 
-    RUNNING_MODES = ['train', 'infer', 'eval']  # Options: 'train', 'infer', 'eval'
-    # Path to save model
-    root = ''
-    # Path to the stored CMMD dataset location
+    RUNNING_MODES = ['train', 'infer', 'eval']
```

Comment:
maybe this should be part of the hydra config as well?

Reply:
accepted